docs/how-to-guides/eagleye-integration-guide.md

```diff
@@ -0,0 +1,140 @@
+# Eagleye Integration Guide
+
+This page will show you how to set up [eagleye](https://github.com/MapIV/eagleye), which is integrated into autoware.
+
+Discussions during the integration are as follows.
+https://github.com/orgs/autowarefoundation/discussions/3257
+
+## What is Eagleye?
+
+Eagleye provides a cost-effective alternative to LiDAR and point cloud-based localization by using low-cost GNSS and IMU sensors to provide vehicle position and attitude.
+Eagleye can also compensate for wheel speed and IMU angular velocity bias by using GNSS Doppler velocity.
+
+Autoware users will be able to choose between their existing LiDAR and point cloud-based localization stacks or GNSS/IMU-based eagleye localizer, depending on their specific needs and operating environment.
+
+There are two ways to utilize eagleye results with the autoware localization stack.
+
+1. Feed only Twist into the EKF localizer.
+
+![eagleye twist integration](images/eagleye-integration-guide/eagleye-twist.png)
+
+2. Feed both Twist and Pose from eagleye into the EKF localizer.(twist can also be used with regular gyro_odometry
+)
+
+![eagleye pose integration](images/eagleye-integration-guide/eagleye-pose.png)
+
+Note that RTK positioning is required for localization using the eagleye pose alone. RTK positioning is not required for twist.
+
+## Behavior when eagleye is executed by autoware
+
+ eagleye estimates are largely based on the following
+
+### Static Estimation
+eagleye needs to be stationary for about 3~5 seconds (yaw_rate_offset_stop.estimated_interval in eagleye_config.yaml) after startup. Static estimation is performed even in environments where GNSS is not received. At this point the yaw rate offset is corrected.
+
+### Movement Estimation
+Next, it needs to travel in a straight line for about 20~30 seconds (heading.estimated_minimum_interval and velocity_scale_factor.estimated_minimum_interval) and the wheel speed scale factor and azimuth angle are estimated. At this point, the estimation of twist is complete and pose will begin to be output.
+
+## eagleye setup
+
+### gnss ros drivers setting
+
+In eagleye, it is necessary to obtain velocity information from GNSS, in addition to NavSatFix.
+
+ex
+ - [ublox_gps](https://github.com/KumarRobotics/ublox/tree/humble-devel/ublox_gps)
+
+This ROS driver publishes sensor_msgs/msg/NavSatFix and geometry_msgs/msg/TwistWithCovarianceStamped required for eagleye with default settings
+
+https://github.com/KumarRobotics/ublox/blob/humble-devel/ublox_msgs/msg/NavPVT.msg
+
+ - [septentrio_gnss_driver](https://github.com/septentrio-gnss/septentrio_gnss_driver/tree/ros2)
+
+Set `publish.navsatfix` and `publish.twist` in the parameter `yaml` file to `true`
+
+https://github.com/septentrio-gnss/septentrio_gnss_driver/blob/ros2/config/gnss.yaml#L90
+
+### eagleye topic setting
+
+You must specify input topics.
```

Comment:
L59-60
```suggestion
You must correctly specify input topics for GNSS latitude, longitude, and height information, GNSS speed information, IMU information, and vehicle speed information in the [`eagleye_config.yaml`](https://github.com/MapIV/autoware_launch/blob/3f04a9dd7bc4a4c49d4ec790e3f6b9958ab822da/autoware_launch/config/localization/eagleye_config.param.yaml#L7-L16).
```

- Will this [commit link](https://github.com/MapIV/autoware_launch/blob/3f04a9dd7bc4a4c49d4ec790e3f6b9958ab822da/autoware_launch/config/localization/eagleye_config.param.yaml#L7-L16) always available in the future? Will the branch be removed once the PRs are merged?